pandas/tests/reshape/test_tile.py

```diff
@@ -104,6 +104,12 @@ def test_cut_corner(self):
 
         pytest.raises(ValueError, cut, [1, 2, 3], 0.5)
 
+    @pytest.mark.parametrize('arg', [2, np.eye(2), DataFrame(np.eye(2))])
```

Comment:
for an existing test, make sure bins are tested for (should work, but covering bases)

scalar, 0-dim ndarray, 1-d ndarray, Series, list, Index

Reply:
Did you mean to test various bins in this specific test (`test_cut_not_1d_arg`)?

Reply:
I mean find a good test (or 2) and make sure that we are exercising all array-likes as input